### PR TITLE
removed genome parameters

### PR DESCRIPTION
**Pull request by KevinMenden**
removed some genome parameters from `nextflow.config` again

### CODE REVIEW

**Comment by KevinMenden** [comment]
Also renamed the genome file in the `bowtie_indices` process such that we don't modify the genome file itself, but rather create a new file.

**Comment by KevinMenden** [comment]
As a hopefully final change, I've added the newest version of `NfcoreSchema.groovy`.
This is from the `dev` branch of tools, and hence it doesn't match the current template so we need to ignore this file during lint checks for now.

File: nextflow.config [hunk 1]
```diff
@@ -48,7 +44,7 @@ params {
   plaintext_email = false
   monochrome_logs = false
   help = false
-  igenomes_base = 's3://ngi-igenomes/igenomes'
+  igenomes_base = 's3://ngi-igenomes/igenomes/'
```
Comment (ewels) [inline nextflow.config:47]: Was this intentional? Pretty sure we removed this trailing slash not that long ago..

**Review by ewels — APPROVED** [review]
LGTM! Just the igenomes slash question.

File: nextflow.config (hunk 1 above)
Comment (KevinMenden) [reply nextflow.config:47]: Yes that was intentional, added this while bug hunting - copied it from the nf-core/rnaseq config.
But it didn't change anything of course so I'll remove again 👍 
Thanks for the review!

Comment (KevinMenden) [inline nextflow.config:47]:
```suggestion
  igenomes_base = 's3://ngi-igenomes/igenomes'
```

Comment (ewels) [reply nextflow.config]: Rnaseq needs fixing then. It breaks something, can't remember what now..

**Pull request merged**